integration/services/ha_backend/init

```diff
@@ -109,8 +126,14 @@ enable = true
 DOC
 
     sleep 45
-    curl -k "http://${ha_backend_container1_ip}:9200"
-    curl -k "http://${ha_backend_container2_ip}:9200"
+    curl -k "https://${ha_backend_container1_ip}:9200" \
+	    --cacert "${ha_backend_container1}/certificates/MyRootCA.pem" \
+	    --key "${ha_backend_container1}/certificates/odfe-admin.key" \
```

Comment:
do we need `--key` and `--cert`? We expected this to change to use basic auth. Also, if you're passing in `-k`, you shouldn't need `--cacert`